__tests__/api.test.js

```diff
@@ -35,4 +35,19 @@ describe('GET /api', () => {
     })
 })
 
-  
\ No newline at end of file
+describe('GET /api/articles/:article_id', () => {
+    test('GET 200: should respond with the articles id', () => {
+        return request(app).get('/api/articles/1').expect(200)
+        .then((response) => {
+            const article = response.body.article
+            expect(article.article_id).toEqual(1)
+        })
+    })
+    test('GET 404: should respond with an error if the article is not found', () => {
+        return request(app).get("/api/articles/999").expect(404)
+        .then(({body}) => {
+            const {msg} = body
+            expect(msg).toBe("Not Found")
+        })
+    })
+})
```

Comment:
There is a missing test here, what would happen if I went to ```/api/articles/mystery``` and what error status should I get?